common/debug.c

```diff
@@ -29,6 +29,7 @@ static void print_hex(int width, bool zero, uint32_t num);
 static void print_padding(int width, bool zero);
 static void print_signed(char sign, int width, bool zero, int32_t num);
 static void print_string(const char *s);
+void run_tests(void);
```

Comment:
This prototype goes in the header (`include/common/test.h`), not here.

Reply:
done